lib/lhm/entangler.rb

```diff
@@ -27,8 +27,8 @@ def initialize(migration, connection = nil)
     def entangle
       [
         create_delete_trigger,
+        create_update_trigger,
         create_insert_trigger,
-        create_update_trigger
```

Comment:
I am assuming the problematic case would be an overkill to replicate in an integration test and unit testing this method makes little sense.

Reply:
I'm not sure how you could replicate it consistently, given its strong dependence on timing. We can start the trigger creation and blast inserts and updates from another thread, but we cannot know what order the operations will actually happen in the database.

There are unit tests for this method that verify the correct creation of all triggers, but none that verify the order. Let me change that.